Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -367,6 +367,10 @@ def _wait_until_device_is_usable(device, deadline):
     @staticmethod
     def _boot_device(device, host=None):
         host = host or SystemHost.get_default()
+
+        # FIXME: remove this workaround after rdar://129789675 has been resolved.
+        host.executive.run_command(['sh', '-c', "mkdir -m 755 -p " + "~/Library/Developer/CoreSimulator/Devices/" + device.udid + "/data/private/var/db"])
```

Comment:
Are these bits correct? One I have on my machine is `700`:

```
drwxr-xr-x  3 ap  staff   96 Mar 27 16:25 MobileAsset
drwx------  5 ap  staff  160 Mar 27 16:25 db
drwx--x--x  3 ap  staff   96 Mar 27 16:25 protected
```

Reply:
That's a good point, I think this can be changed to 700.

Thanks for reviewing!